CensusIncomePrediction.ipynb

```diff
@@ -113,9 +109,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "eval_config = bolt.EvalConfig().with_metrics([\"categorical_accuracy\"])\n",
-    "\n",
-    "model.evaluate(test_filename, eval_config);"
+    "model.evaluate(test_filename, metrics=[\"categorical_accuracy\"]);"
```

Comment:
Minor: The semicolon is not needed (I think this was there in a couple of the notebooks)